api/src/main/java/org/openmrs/module/m2sysbiometrics/util/impl/M2SysHelperImpl.java

```diff
@@ -0,0 +1,24 @@
+package org.openmrs.module.m2sysbiometrics.util.impl;
+
+import org.apache.commons.lang.StringUtils;
+import org.openmrs.api.APIException;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.module.m2sysbiometrics.util.M2SysHelper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class M2SysHelperImpl implements M2SysHelper {
```

Comment:
would be good to move some from other classes here since we already have a class like this - make this the central place to retrieve all M2SysProperties

Reply:
Do you mean, that I should create methods for getting all properties used in the M2Sys module or just stay it as is, but use getGlobalProperty everywhere in the module?

Reply:
I meant using this as the go to class for properties, avoiding direct usage of adminService